app/screens/browse_channels/channel_dropdown.tsx

```diff
@@ -90,13 +90,13 @@ export default function ChannelDropdown({
     }
     return (
         <View
-            testID='browse_channels.channel.dropdown'
+            testID='browse_channels.channel_dropdown'
```

Comment:
this can be one line `<View testID='browse_channels.channel_dropdown'>`

Reply:
Updated.